app/assignment/directives/courseAssignmentTable.js

```diff
@@ -190,7 +190,7 @@ let courseAssignmentTable = function ($rootScope, AssignmentActionCreators) {
 								courseHtml += "</div>";
 
 								courseHtml += "<div class=\"course-units\">";
-								courseHtml += "Units: " + course.unitsLow;
+								course.unitsHigh > 0 ? courseHtml += "Units: " + course.unitsLow + " - " + course.unitsHigh : courseHtml += "Units: " + course.unitsLow;
```

Comment:
We mess up this logic a lot, so I would instead add a transient helper on the entity in the backend, and then make use of that here.